src/pages/Lend/LoanSearchPage.vue

```diff
@@ -1,34 +1,26 @@
 <template>
-	<www-page
-		id="lend-filter-alpha"
-	>
+	<www-page id="lend-filter-alpha">
 		<article class="tw-bg-secondary tw-relative tw-pt-6">
-			<div class="tw-relative" style="max-height: 460px;">
-				<div class="tw-absolute tw-top-0 tw-h-full tw-w-full tw-overflow-hidden">
-				</div>
-			</div>
-			<!-- <div class="lg:tw-absolute lg:tw-w-full lg:tw-h-full lg:tw-top-0 lg:tw-pt-8">
-			</div> -->
 			<div>
 				<kv-page-container>
-					<div class="tw-flex tw-items-start">
+					<div class="tw-flex tw-items-start tw-pb-8">
 						<div class="tw-flex-1">
-							<h2 class="tw-mb-2">
+							<h1 class="tw-mb-2">
 								Make a loan, change a life
-							</h2>
-							<p class="tw-mb-3p tw-hidden md:tw-block">
+							</h1>
+							<span class="better-future md:tw-w-2/3 tw-mb-3p tw-hidden md:tw-block">
```

Comment:
I think this should remain a `p` tag and use the `tw-text-h3` class along with `tw-text-subhead`